src/test/java/uk/gov/hmcts/reform/sscs/validators/SscsCaseValidatorTest.java

```diff
@@ -714,8 +715,9 @@ public void givenAnAppointeeDoesNotContainAnAddressLine3_thenAddAWarning() {
     }
 
     @Test
-    public void givenAnAppointeeDoesNotContainAnAddressLine4_thenAddAWarning() {
+    public void givenAnAppointeeHasAddressAll4LinesAndDoesNotContainAnAddressLine4_thenAddAWarning() {
```

Comment:
This test method name is confusing me - what is it we want to achieve?

Reply:
ok, going to rename